zips/zip-0315.rst

```diff
@@ -209,6 +241,10 @@ respectively. The following confirmation policy is RECOMMENDED:
 * 10 confirmations, for untrusted TXOs;
 * 3 confirmations, for trusted TXOs.
 
+Wallets MUST NOT permit the spending of TXOs with fewer than 3 confirmations,
```

Comment:
Doesn't this conflict with sending to a TEX address?

Reply:
Ah, yeah, TEX secondary transactions need to be exempted explicitly.